recipes/PyFstat/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "PyFstat" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5275804036a0d814acf111857afd46fbb3cba81e48f3d5275fbdd7219d75dddc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >3.6,<3.9
```

Do you really need to prevent installations on Python 3.9?
```suggestion
    - python >3.6
```

---

One of our main dependencies (lalsuite) hasn't been released for 3.9 yet. So in a little while we should be good to remove the constraint, but not just yet. Bad timing for submitting this request, I guess, sorry!

---

@duncanmmacleod suggested to just remove the constraint already, as when people try to install `PyFstat` for `python 3.9` there will be a clear enough message that it's the dependencies that are missing.